Automattic/wp-calypso
# Add a post-purchase redirect parameter for Premium themes upgrade nudge

#### Proposed Changes

* Add a redirect_to parameter to logged-in upgrade nudges that are displayed in the Theme details page for Premium themes.

#### Testing Instructions



* Checkout the branch locally on your Calypso environment (the Calypso.live environment has an issue with redirect_to parameters)
* Using a free site, navigate to theme/alonso/your-site
* Click on the upgrade nudge and toggle between Yearly and Monthly tabs
* Notice that the `redirect_to` parameter is present between toggles
* Purchase a Premium subscription
* After you've purchased the subscription, you should be automatically redirected to `theme/alonso/your-site`
* Check that a notice appears on the top right of the page mentioning that the purchase has been made

#### Pre-merge Checklist





Related to https://github.com/Automattic/dotcom-forge/issues/1456
